docs/tutorial/index.rst

```diff
@@ -0,0 +1,62 @@
+Javascript Sandbox Tutorial
+===========================
+
+This is a javascript sandbox tutorial for writing Vumi Go Application.
+
+What is sandbox?
+----------------
+
+A sandbox is a type of software testing environment that enables the isolated execution of software or programs for independent evaluation, monitoring or testing. Basically sandboxing is to test a software in a controlled environment before putting it to actual use.
+
+
+Introduction to an example we’re going to use for this tutorial
+---------------------------------------------------------------
+
+
+Outcomes of the tutorial
+------------------------
+
+By the end of this tutorial, you will be able to:
+
+- Know how to use Sandbox skeleton
```

Comment:
I would say "Understand the structure of a sandbox application repository" (the reader might not know what the skeleton is at this point).